SECURITY_CHARTER.md

```diff
@@ -0,0 +1,81 @@
+* Coordination: overseeing the response to reported vulnerabilities to ensure compliance with SLA deadlines.  
+* Process improvement: maintaining and enhancing security measures, such as implementing linters, scanners, fuzzers, and patch managers. Ensuring security is proactively integrated throughout the project.
+
+#### Rotating shifts
+
+* Team members take turns being the primary point of contact on a weekly basis.  
+* The designated person on the shift handles incoming security requests, coordinates responses to incidents, and manages day-to-day security tasks during their shift.   
+* Other team members will continue to work on security response duties, supporting the person on the shift.  
+* The Keycloak Security Office weekly meeting hours determine the end of the shift, and the next person on the shift is updated about the status.  
+* Vacations and PTOs are communicated during the meeting so we can adjust the shift.
+
+### Keycloak Security Resolution Team
+
+Dynamic teams formed by individuals actively involved in triaging or resolving open CVEs. Members are added when they engage with a vulnerability and removed once their involvement concludes.
+
+### Scope
+
+* Resolution and testing: ensuring vulnerabilities are effectively fixed and thoroughly tested.  
+* Collaboration:  working with the Response team to prioritize fixes above all other items in the team's backlog, regardless of their nature.  
+* Release Coordination: collaborating closely with release coordinators and Quality Engineering (QE) teams to include patches in upcoming releases.
+
+## Access
+
+| Resource | Response Team | Resolution team |
+| :---- | :---- | :---- |
+| [Mailing list](https://groups.google.com/g/keycloak-security) | Full access | Added in CC to specific threads |
+| [Private GitHub repository](https://github.com/keycloak/keycloak-private/) | Full access | Temporary access |
+| [Security advisories and alerts](https://github.com/keycloak/keycloak/security) | Full access | No access |
+| Slack channel (\#alerts-keycloak-cve) | Full access | Temporary access |
+
+## Coordinating a Security Vulnerability Fix
+
+* Identification: the Response Team identifies relevant engineers from affected areas and temporarily includes them in private communication channels (e.g., repositories, email threads), forming a temporary Resolution Team.  
+* Efficiency: to prevent accidental disclosure, the Resolution Team remains as small as necessary.  
+* Autonomy: the Resolution Team has the autonomy to involve additional parties such as release coordinators, QE, and documentation teams. Communication with the Response Team is advised when in doubt.  
+* Access Revocation: post-release, access to sensitive communication channels is revoked to uphold the principle of least privilege.
+
+## Process Overview
+
+1. A new vulnerability is reported to the Keycloak security mailing list.  
+2. A CVE ID is assigned.  
```

Comment:
This should only happen after triage by the Security Response Team. I guess it's implicit here, by might be good to mention.

Reply:
That's implicit, but no problem about explicitly mentioning it.